generators/README.md

```diff
@@ -0,0 +1,17 @@
+# User Data Generator
+
+generate_users_json.py generates a set of users for the Retail Demo Store.
+
+These user profiles are used in the following ways:
+
+* The Users service provides login services to the user profiles that this crates for the Retail Demo Store
```

Comment:
Minor stuff:

- Hyperlink 'Users service' to the Users source directory (../src/users). I'm trying to do that throughout. As more documentation is added for services, linking to them will help reader better understand/navigate the project.
- Type: 'creates' rather than 'crates'